base/client.jl

```diff
@@ -4,25 +4,39 @@
 ##             and REPL
 
 const text_colors = AnyDict(
-    :black   => "\033[1m\033[30m",
-    :red     => "\033[1m\033[31m",
-    :green   => "\033[1m\033[32m",
-    :yellow  => "\033[1m\033[33m",
-    :blue    => "\033[1m\033[34m",
-    :magenta => "\033[1m\033[35m",
-    :cyan    => "\033[1m\033[36m",
-    :white   => "\033[1m\033[37m",
-    :normal  => "\033[0m",
-    :bold    => "\033[1m",
+    :black         => "\033[30m",
+    :red           => "\033[31m",
+    :green         => "\033[32m",
+    :yellow        => "\033[33m",
+    :blue          => "\033[34m",
+    :magenta       => "\033[35m",
+    :cyan          => "\033[36m",
+    :white         => "\033[37m",
+    :light_black   => "\033[90m",
```

Comment:
I don't think I've ever heard gray called light black, but I guess it works. throw a comment in?